web/front/src/styles/_var.sass

```diff
@@ -75,6 +75,7 @@ $icon-size: 40px
 
 // MARGINS, PADDINGS VARS
 $border-radius: 5px
+$gap: 5px
```

Comment:
rename to gap-xxs if necessary but the var is only used once in that situation you can use mathematical operator in your Sass such as ` var.$gap-xs/4`